app/uk/gov/hmrc/economiccrimelevyregistration/models/AmlSupervisorType.scala

```diff
@@ -53,7 +53,7 @@ object AmlSupervisorType {
   ): Seq[RadioItem] = {
     val amlProfessionalBodySupervisorOptions: Seq[SelectItem] =
       SelectItem(
-        text = ""
+        text = "&nbsp;"
```

Comment:
This is just a hack you may like to merge in.
Read the following issue if you wish to understand more:
https://github.com/hmrc/play-frontend-hmrc/issues/223